src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -157,6 +157,20 @@ public void testNonSpecifiedImports() throws Exception {
         verify(checkConfig, getPath("InputCustomImportOrderDefault.java"), expected);
     }
 
+    @Test
+    public void testOrderRuleEmpty() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(CustomImportOrderCheck.class);
+        checkConfig.addAttribute("customImportOrderRules", "");
+        final String[] expected = {
+            "7: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "java.util.List"),
+            "13: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "com.puppycrawl.tools.*"),
+            "16: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "com.google.common.base.*"),
+        };
+
+        verify(checkConfig, getPath("InputCustomImportOrderDefault2.java"), expected);
```

Comment:
we try to force us to create Input file for each test and place config in Input file.
example: https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/InputJavadocContentLocationFirstLine.java#L3

Reply:
Will do

Reply:
please do not reply anything to review item except for "done" or any other clarification.
you will do this fores sure, I need clear signal when you are ready for review.

Reply:
Sorry about that, was hoping to get clarification on the earlier point before pushing everything together.

Fix has been pushed and ready for review.